os/fs/smartfs/smartfs.h

```diff
@@ -171,6 +171,13 @@
  * bits from the erased to non-erased state.
  */
 
+#ifdef CONFIG_SMART_FS_SECURE
+#define MAXBUFSIZE 256
+#define CRYPTSIZE 16
+#define KEYSIZE 128
+#define CRYPT_KEY "12345678"
```

Comment:
Hard coded cypto key? what is purpose?

Reply:
Yes it's hard coded as Same key will be used for encryption and decryption through out the life time of data which we are intending to keep secure.  This of course  needs to make it as dynamic ( runtime ) in future and for this commit, we can go ahead with hard coded keys.